server/app/policies/library_entry_policy.rb

```diff
@@ -10,10 +10,13 @@ def show?
     true
   end
 
+  def create?
+    !user.nil?
+  end
```

Comment:
Not sure it's possible to fix, perhaps we can send this upstream? It makes sense IMO to fill the fields before you call the authorizer